core/src/main/scala/akka/persistence/jdbc/testkit/internal/SchemaUtilsImpl.scala

```diff
@@ -114,23 +118,27 @@ private[jdbc] object SchemaUtilsImpl {
     }
   }
 
-  private def dropScriptFor(schemaType: SchemaType): (String, String) =
+  private def dropScriptFor(schemaType: SchemaType, legacy: Boolean): (String, String) = {
+    val suffix = if (legacy) "" else "-v5"
```

Comment:
I think we should invert this. Default one is the new and we add `legacy` or `v4` to previous one.

Reply:
👍